_posts/2025-11-30-update.md

```diff
@@ -41,4 +41,4 @@ slug: 2025-11-30-update
 
 ## Shoutouts
 
-*
+* Petr Mullar -- Shoutout for organizing a meeting to support new contributors in Prow, gathering ideas to improve onboarding and reduce entry barriers for newcomers. 
```

Comment:
So usually we don't rephrase these; we just copy the shout-out as written, except for truncating it if it's too long.